.github/workflows/flatpak.yml

```diff
@@ -3,10 +3,10 @@ name: Flatpak (Experimental)
 on:
   push:
     paths-ignore: ['**.md']
-    branches: [master]
+    branches: [disabled-master]
   pull_request:
     paths-ignore: ['**.md']
-    branches: [master]
+    branches: [disabled-master]
```

Comment:
Just why ?

Reply:
Because flatpak is part of the main workflow already (as it's based on the cmake-overhaul).